config/secrets.yml

```diff
@@ -34,8 +34,8 @@ development:
   <<: *default
   secret_key_base: asjdahkjda
   devise: abc123
-  chargify_app_name: loomio-test
-  chargify_api_key: fakeapikey
+  chargify_app_name: <%= ENV['CHARGIFY_APP_NAME'] %>
```

Comment:
You can actually just take those two lines out; we'll inherit the ENV thingies from the default block above